app/src/main/res/values/shape.xml

```diff
@@ -8,4 +8,14 @@
     <item name="cornerSizeBottomLeft">@dimen/spacing_0</item>
     <item name="cornerSizeBottomRight">@dimen/spacing_0</item>
   </style>
+
+  <style name="ShapeAppearanceOverlay.Clinic.V2.UnitPreferenceButton" parent="">
+    <item name="cornerFamily">rounded</item>
+    <item name="cornerSize">50%</item>
+  </style>
+
+  <style name="Clinic.V2.MaterialButton.UnitPreference">
+    <item name="cornerRadius">@null</item>
```

Comment:
Also, it seems like you forgot to set the min-height for this button, can we also set the text appearance in this style instead of at the view, please? Please check the mockups or we can pair on this if you need more info.

Reply:
The current button height in the mockups is 32dp. The current default button height in Simple is 48dp, that's why the button looks a bit big compared to mockups.